module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -446,14 +446,27 @@ protected function getAvailabilityMessage(AvailabilityStatusInterface $availabil
      */
     protected function renderFullStatus($record, $simpleStatus, array $values = [])
     {
+        // Default case: no extra holdings fields are shown
+        $holdingsTextFieldsToShow = [];
+
+        if ($this->config->Item_Status->include_holdings_text_fields ?? false) {
+            // If we are showing additional holdings text fields, the set of fields shown is
+            // either config.ini's displayed_holdings_text_fields[] (if set), or the set of
+            // all fields reported by the ILS driver otherwise.
+            $holdingsTextFieldsToShow = $this->config?->Item_Status?->displayed_holdings_text_fields?->toArray()
```

Comment:
I adjusted this slightly to make it more concise and to ensure that `$holdingsTextFieldsToShow` is always an array, rather than sometimes a `Laminas\Config\Config` object.

Reply:
Thanks!  I noticed the `Config` object, but wasn't sure if the cast was desirable.